packages/components/src/components/popover/popover.jsx

```diff
@@ -89,6 +90,9 @@ class Popover extends React.PureComponent {
                         transitionDuration={0.25}
                         padding={margin + 8}
                         contentDestination={relative_render ? this.state.popover_ref : document.body}
+                        containerClassName={classNames({
+                            'react-tiny-popover-container_disabled_pointer_event': should_pointer_event_disabled,
```

Comment:
- Since `disabled_pointer_event` is a modifier of the original element, could we prefix it with `--`

```suggestion
                            'react-tiny-popover-container--disabled-pointer-event': should_disable_pointer_events,
```